stow/sass/_full-site-editing.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../varia/sass/full-site-editing/imports";
+
+$spacing_vertical: map-deep-get($config-global, "spacing", "vertical");
+
+.fse-enabled.home.page.hide-homepage-title .fse-header.entry-content {
+	padding-bottom: $spacing_vertical;
+	@include media(mobile) {
+		padding-bottom: #{1.5 * $spacing_vertical};
+	}
+}
+
+.fse-enabled {
+	#masthead {
+		.site-description {
+			color: unset;
+		}
+		.site-title {
+			font-size: unset;
+		}
+	}
+}
```

Comment:
These also unset the blocks style attributes.
Unfortunately, this is one of those annoying cases where we need to work outside the FSE files, and into the original theme:

https://github.com/Automattic/themes/blob/3834391965040259a8fe546a375153b385ec59dc/stow/sass/_extra-child-theme.scss#L60-L65

Moving those rules into something like (right after the `#masthead` block) this would suffice:

```scss
// Prevent overruling the user defined font size and color set inside Gutenberg
// for Full Site Editing's Site Title and Site Description blocks.
body:not( .fse-enabled ) #masthead {
	.site-title {
		font-size: #{map-deep-get($config-global, "font", "size", "xxl")};
	}
	.site-description {
		color: #333;
	}
}
```

Reply:
Oh, just noticed that this happens for the navigation menu as well.
Please replace the unsets with a `body:not( .fse-enabled )` where needed.

Reply:
Damn, I was pretty sure the block attributes were still working :/